lightning/src/routing/router.rs

```diff
@@ -542,7 +558,7 @@ where L::Target: Logger {
 
 pub(crate) fn get_route<L: Deref, S: Score>(
 	our_node_pubkey: &PublicKey, payee: &Payee, network: &NetworkGraph,
-	first_hops: Option<&[&ChannelDetails]>, final_value_msat: u64, final_cltv_expiry_delta: u32,
+	first_hops: Option<&[&ChannelDetails]>, final_value_msat: u64, final_cltv_expiry_delta: u32, 
```

Comment:
nit: remove trailing space

Reply:
Done in https://github.com/lightningdevkit/rust-lightning/pull/1234/commits/ebecbddf9c7375ddea65b0263e959b60eb308b6d.